src/main/java/org/apache/sysds/runtime/instructions/cp/ReorgCPInstruction.java

```diff
@@ -83,6 +85,31 @@ private ReorgCPInstruction(Operator op, CPOperand in, CPOperand out, CPOperand c
 		_col = col;
 		_desc = desc;
 		_ixret = ixret;
+		_shift = new CPOperand();
+	}
+
+	/**
+	 * for opcode roll
+	 *
+	 * @param op
+	 *            operator
+	 * @param in
+	 *            cp input operand
+	 * @param shift
+	 *            ?
+	 * @param out
+	 *            cp output operand
+	 * @param opcode
+	 *            the opcode
+	 * @param istr
+	 *            ?
+	 */
```

Comment:
remove docs if they are empty like this.

Reply:
I filled in the blanks in the comment.

Reply:
the formatting is off then, here is a correctly formatted one:

```java
	/**
	 * Apply binary row operation on the right side with one extra row evaluating with zeros.
	 * 
	 * @param op         The operation to this dictionary
	 * @param v          The values to use on the right hand side.
	 * @param colIndexes The column indexes to consider inside v.
	 * @return A new dictionary containing the updated values.
	 */
	public IDictionary binOpRightAndAppend(BinaryOperator op, double[] v, IColIndex colIndexes);
```

Reply:
I reformatted all the code written in this PR and the code in `ReorgCPInstruction.java` that didn’t match the comment format using the IDE.